src/main/java/cuchaz/enigma/translation/representation/entry/ParentedEntry.java

```diff
@@ -51,6 +53,12 @@ public P getParent() {
 		return parent;
 	}
 
+	@Nullable
+	@Override
+	public String getJavadocs() {
```

Comment:
I'm not sure it makes much sense to attach javadocs to the entries like this -- imo that should stay just apart of the mapping. All the places where this is referenced could just check the mapping instead?